Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = ["gui"]
 rust-version = "1.65.0"
 
 [package.metadata.docs.rs]
-features = ["serde"]
+features = ["rwh_04", "rwh_05", "rwh_06", "serde"]
```

Comment:
Could we drop `04` while at it?

Reply:
`raw-window-handle` v0.4 is still in use by some users, so I think we should keep it for now.